internal/pkg/config/config.go

```diff
@@ -283,7 +284,7 @@ func (fm funcMatcher) MatchFunction(path, receiver, name string) bool {
 	return fm.Package.MatchString(path) && fm.Receiver.MatchString(receiver) && fm.Method.MatchString(name)
 }
 
-var readFileOnce sync.Once
+var readFileOnce *sync.Once = new(sync.Once)
```

Comment:
This change does not seem to belong in this PR?

Reply:
It was necessary so that the tests could read another config. But no needed any more.